src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -31,6 +31,15 @@ const actions: ActionTree<IEmptyState, IRootState> = {
         ? ({...choice, name: resourceValue.value})
         : choice
       ))
+
+    const propertyValueMapping = block.vendor_metadata?.floip?.ui_metadata?.set_contact_property?.property_value_mapping
+    if (propertyValueMapping !== undefined) {
```

Comment:
Could we add a comment above this to explain WHY ? It's not obvious at the 1st glance and might takes time to understand this